### ISSUE TITLE
RCV test

### ISSUE THREAD

**Issue by davidozog** [davidozog/openshmem-specification#5]
<form method="post"
      action="https://civs.cs.cornell.edu/cgi-bin/vote.pl"
      enctype="multipart/form-data"
      name="CastVote">
<input type="hidden" name="key" value="80b39f6b9526bf9f"  />
<input type="hidden" name="id" value="E_ba7951d5c1c6f16a"  />
<input type="hidden" name="akey" value="fd4eaaf72e367c89"  />
<table class="form" id="ballot" border="0" cellpadding="5" cellspacing="0"><tr><td>
<table cellpadding="5px" cellspacing="0" border="1" id="preftable">
    <tr class="heading">
	    <th> Choice </th><th>Rank</th></tr>
<tr onclick="select_row(this, event.shiftKey||event.ctrlKey);"><td class="choice">6</td>
<td><select size="1" name="C6" onchange="sort_rows()">
  <option value="1" label="1st" >1st</option>
  <option value="2" label="2nd" >2nd</option>
  <option value="3" label="3rd" >3rd</option>
  <option value="4" label="4th" >4th</option>
  <option value="5" label="5th" >5th</option>
  <option value="6" label="6th" >6th</option>
  <option value="7" label="7th"  selected="selected">7th</option>
</select></td>
</tr>
<tr onclick="select_row(this, event.shiftKey||event.ctrlKey);"><td class="choice">2</td>
<td><select size="1" name="C2" onchange="sort_rows()">
  <option value="1" label="1st" >1st</option>
  <option value="2" label="2nd" >2nd</option>
  <option value="3" label="3rd" >3rd</option>
  <option value="4" label="4th" >4th</option>
  <option value="5" label="5th" >5th</option>
  <option value="6" label="6th" >6th</option>
  <option value="7" label="7th"  selected="selected">7th</option>
</select></td>
</tr>
<tr onclick="select_row(this, event.shiftKey||event.ctrlKey);"><td class="choice">4</td>
<td><select size="1" name="C4" onchange="sort_rows()">
  <option value="1" label="1st" >1st</option>
  <option value="2" label="2nd" >2nd</option>
  <option value="3" label="3rd" >3rd</option>
  <option value="4" label="4th" >4th</option>
  <option value="5" label="5th" >5th</option>
  <option value="6" label="6th" >6th</option>
  <option value="7" label="7th"  selected="selected">7th</option>
</select></td>
</tr>
<tr onclick="select_row(this, event.shiftKey||event.ctrlKey);"><td class="choice">1(b)</td>
<td><select size="1" name="C1" onchange="sort_rows()">
  <option value="1" label="1st" >1st</option>
  <option value="2" label="2nd" >2nd</option>
  <option value="3" label="3rd" >3rd</option>
  <option value="4" label="4th" >4th</option>
  <option value="5" label="5th" >5th</option>
  <option value="6" label="6th" >6th</option>
  <option value="7" label="7th"  selected="selected">7th</option>
</select></td>
</tr>
<tr onclick="select_row(this, event.shiftKey||event.ctrlKey);"><td class="choice">5</td>
<td><select size="1" name="C5" onchange="sort_rows()">
  <option value="1" label="1st" >1st</option>
  <option value="2" label="2nd" >2nd</option>
  <option value="3" label="3rd" >3rd</option>
  <option value="4" label="4th" >4th</option>
  <option value="5" label="5th" >5th</option>
  <option value="6" label="6th" >6th</option>
  <option value="7" label="7th"  selected="selected">7th</option>
</select></td>
</tr>
<tr onclick="select_row(this, event.shiftKey||event.ctrlKey);"><td class="choice">3</td>
<td><select size="1" name="C3" onchange="sort_rows()">
  <option value="1" label="1st" >1st</option>
  <option value="2" label="2nd" >2nd</option>
  <option value="3" label="3rd" >3rd</option>
  <option value="4" label="4th" >4th</option>
  <option value="5" label="5th" >5th</option>
  <option value="6" label="6th" >6th</option>
  <option value="7" label="7th"  selected="selected">7th</option>
</select></td>
</tr>
<tr onclick="select_row(this, event.shiftKey||event.ctrlKey);"><td class="choice">1(a)</td>
<td><select size="1" name="C0" onchange="sort_rows()">
  <option value="1" label="1st" >1st</option>
  <option value="2" label="2nd" >2nd</option>
  <option value="3" label="3rd" >3rd</option>
  <option value="4" label="4th" >4th</option>
  <option value="5" label="5th" >5th</option>
  <option value="6" label="6th" >6th</option>
  <option value="7" label="7th"  selected="selected">7th</option>
</select></td>
</tr>
</table></td>
</tr>
<tr><td style="height: 100%"><input id="vote" type="submit" value="Submit ranking" name="Vote" /></td></tr>
</table>
</form>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.4.1/jquery.min.js"></script>
<script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.7.2/jquery-ui.min.js"></script>
<script src="/vote.js"></script>
<script>setup()</script>
